npm.js

```diff
@@ -66,6 +73,8 @@ exports.translate = function(load){
 				if(pkg.browser){
 					delete pkg.browser.transform;
 				}
+				pkg = transform(loader, load, pkg);
```

Comment:
> but i also dont know why we need the jsonTransformation in this situation (for fetching the plackages)? why is a custom tranform method needed in this situtation?

Depends what you mean?

If you mean why doesn't the system-json extension handle this situation it's because these json files are not imported (they don't go through the loader hooks).

If you mean why is this feature needed? I don't think that it is needed, you are the one that wanted to add it :)